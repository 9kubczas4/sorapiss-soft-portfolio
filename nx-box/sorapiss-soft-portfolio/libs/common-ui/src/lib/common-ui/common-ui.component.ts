import { ChangeDetectionStrategy, Component } from '@angular/core';
import { CommonModule } from '@angular/common';

@Component({
  selector: 'ssp-ui-common-ui',
  standalone: true,
  imports: [CommonModule],
  templateUrl: './common-ui.component.html',
  styleUrls: ['./common-ui.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush,
})
export class CommonUiComponent {}
