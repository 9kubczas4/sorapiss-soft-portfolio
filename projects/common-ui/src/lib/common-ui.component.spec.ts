import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
import { CommonUiComponent } from './common-ui.component';

describe('NgUtilsComponent', () => {
  let spectator: Spectator<CommonUiComponent>;
  const createComponent = createComponentFactory({
    component: CommonUiComponent,
    shallow: true,
  });

  beforeEach(() => {
    spectator = createComponent();
  });

  it('should match snapshot', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
