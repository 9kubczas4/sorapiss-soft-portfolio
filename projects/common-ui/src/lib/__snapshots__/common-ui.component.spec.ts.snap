// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NgUtilsComponent should match snapshot 1`] = `
<lib-common-ui
  __ngContext__={[Function LRootView]}
>
  <p>
     common-ui works! 
  </p>
</lib-common-ui>
`;

exports[`NgUtilsComponent should match snapshot when diagram is loading 1`] = `
<lib-ng-utils
  __ngContext__={[Function LRootView]}
>
  <p>
    ng-utils component works!
  </p>
</lib-ng-utils>
`;
